### PR TITLE
feat: added txt file parser and printer

### PR DESCRIPTION
**Pull request by AsafIfergan**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from utils import ParseFiles
+import argparse
+
+parser = argparse.ArgumentParser(description='Parse files')
+parser.add_argument('-f', '--file', help='Path to a file', required=True)
+args = parser.parse_args()
+
+txtfile = ParseFiles(args.file)
+txtfile.parse_text_files()
```
Comment (nodeav) [inline main.py:9]:
```suggestion
parser = ParseFiles(args.file)
parser.parse_text_files()
```

File: utils.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+class ParseFiles:
+
+    def __init__(self, file):
+        self.file = file
+
+    def parse_text_files(self):
```
Comment (nodeav) [inline utils.py:7]: 1. files? it's just 1 file, right?
2. `parse_text_file` in a class named `ParseFiles` seems redundant. Change to `parse`.
     just like you wouldn't have a class named `Dog` with a method named `dog_bark`

File: utils.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+class ParseFiles:
```
Comment (nodeav) [inline utils.py:2]: Classes represent objects, not actions. Thus, a more correct name would be something like `FileParser`.
Since this is used for `txt` files specifically, I'd call it `TxtFileParser` or `KeyValueFileParser`, or whatever naming convention we're going to go with for this type of files.

File: main.py (hunk 1 above)
Comment (nodeav) [inline main.py:9]: missing newline at end of file, run `reformat code`

File: utils.py (hunk 3 above)
Comment (AsafIfergan) [reply utils.py:2]: I think "FileParser" would be the correct choice since our next task (or one of the next few ones) is to build parsing functions for other filetypes. What do you think?

File: utils.py (hunk 2 above)
Comment (AsafIfergan) [reply utils.py:7]: That's right. Thanks for the clarifications.

File: main.py (hunk 1 above)
Comment (AsafIfergan) [reply main.py:9]: Cool.

File: utils.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+    def __init__(self, file):
+        self.file = file
+
+    def text_file(self):
+        with open(self.file) as f:
+            for line in range(5):
+                split_file = f.readline().split(":")
```
Comment (nodeav) [inline utils.py:8]: Does not handle edge-cases; for an example, try adding `fake@email.com:` to the test file. 

I think a better way would be:
```python
email, _, password = f.readline().partition(":")
```

because `partition` _always_ returns a 3-tuple (split can return any number >= 1).

Comment (AsafIfergan) [reply utils.py:8]: That's really cool.

File: utils.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class TxtFileParser:
+    def __init__(self, file):
+        self.file = file
+
+    def text_file(self):
```
Comment (nodeav) [inline utils.py:5]: Rename to parse!

**Review by nodeav — APPROVED** [review]
